### ISSUE TITLE
[Request] - Consider adding (view-source:) to the list

### ISSUE THREAD

**Issue by tigerjack** [overdodactyl/userContent-tweaks#7]
The title says pretty much everything. It could be useful to have the dark theme also on `view-source` pages. 

Btw, thanks for the great work you've done. I've tried to make simple tweaks in the past, but I'm not much of a css expert.

**Comment by overdodactyl** [comment]
Hi @tigerjack, I'm glad you like the project!

Just to be clear, you're referring to window that pops up when choosing "View Page Source" from the context menu, or something different?

**Comment by tigerjack** [comment]
@overdodactyl this was exactly what I mean. For the moment, I've just added `url-prefix(view-source:)` somewhere in the `userContent.css`.

**Comment by overdodactyl** [comment]
@tigerjack, when you have the chance, can you check out the new styling for `view-source:` pages?

I admittedly don't use them very often, and I didn't do any extensive testing with the dark style.  

If you run across any issues or find any font coloration to be too hard to read, please let me know.

**Comment by tigerjack** [comment]
@overdodactyl thanks for the effort. It seems almost good to me. However the violet (magenta?) color in the end tag seems quite uncomfortable to me; also, it doesn't get applied to the start tag, but just to the end tag. Is it the expected behavior?

Here is a picture extracted from a web page source. 
![image](https://user-images.githubusercontent.com/8199216/33707548-0e1342d6-db38-11e7-85b4-a29ee87b04ea.png)

**Comment by overdodactyl** [comment]
@tigerjack, thanks for the feedback....much appreciated.

Instead of my own crazy concoction of colors, I decided it might be better to use the Photon colors built into the web-developer tools for the dark theme. Does this look a little more appealing?

![view-source](https://user-images.githubusercontent.com/33071129/33726897-4e4adb30-db6e-11e7-8742-d995ed000bce.png)

I think the start/end tag issue should be resolved too.

**Comment by tigerjack** [comment]
@overdodactyl definitely. Once again, thanks for your great work.

**Comment by overdodactyl** [comment]
No problem!

Any other ideas for improvements/preferences for change here or should we consider this issue closed?

**Comment by tigerjack** [comment]
@overdodactyl I've just changed the color for the attribute field to green (`#00FF00`), just to highlight them. It seems useful to me, but maybe there's a better color defined in your files somewhere else.
Apart from that, you can close the issue.
![image](https://user-images.githubusercontent.com/8199216/33735311-3c33f366-db8f-11e7-88fa-297881c1c08e.png)

**Comment by overdodactyl** [comment]
@tigerjack, I think that's a good idea - what do you think of this shade of green? I think it matches the tone of the other colors a tad better:

![attribut-color](https://user-images.githubusercontent.com/33071129/33735729-0ffe9708-db88-11e7-9e7e-0c9c76c04c46.png)

**Comment by tigerjack** [comment]
@overdodactyl looks perfect 👍

**Comment by overdodactyl** [comment]
One last change for the red and orange colors that were a bit jarring:

![orange](https://user-images.githubusercontent.com/33071129/33736532-d771ddd4-db8a-11e7-91ff-9500a8d5e0e2.png)
![red](https://user-images.githubusercontent.com/33071129/33736537-da382956-db8a-11e7-8a33-94fa47504397.png)

**Comment by tigerjack** [comment]
Seems perfect 👍

**Comment by tukusejssirs** [comment]
Sorry to comment on an old issue, but I don’t think I should open a new one.

I use FF 74.0a1 (2020-01-23) (64-bit) on CentOS 8. I installed ShadowFox:
- quit Firefox;
- download the installer;
- run the installer;
- selected the profile I want to use;
- kept unchecked _Auto-Generate UUIDs_ (anyway, what is it for?);
- checked _Set Firefox dark theme_;
- pressed _Install/Update ShadowFox_;
- started Firefox with the particular profile.

But I cannot see a difference. Particularly, the `view-source` does not work.

Is there anything I need to do? Do I need to enable ShadowFox somewhere? I have followed the [instructions](https://overdodactyl.github.io/ShadowFox/#) (i.e. I have had already dark theme enabled).
